cmake/AddSpectreExecutable.cmake

```diff
@@ -33,6 +33,12 @@ function(add_spectre_executable TARGET_NAME)
     RULE_LAUNCH_LINK "${CMAKE_BINARY_DIR}/tmp/WrapExecutableLinker.sh"
```

Comment:
Do we need to squash more commits to have each individual commit build successfully? It's really annoying to have intermediate commits not build when using `git bisect`.

Reply:
The commits all build individually on my machine.